content/post/data-warehouse-sla-p2.md

```diff
@@ -27,47 +28,61 @@ Here are four examples of services provided with increasing specificity:
 
+In addition to the scope of the problem, it is also helpful to categorize data issues in one of two buckets: stale data or wrong data. An ETL issue may result in stale data in a report sent to everyone in the company. From the perspective of number of people impacted, this would be pretty high severity. However, if everyone is aware that they are seeing yesterday's data there should be less of an issue. The real issue is when your stakeholders use incorrect data and don't know it.  
 
-The most simple problem - stale data, always makes me think of this quote from the late comedian Mitch Hedberg:
+Whenever I encounter a stale data issue, I, always think of this quote from the late comedian Mitch Hedberg:
 
 >"Escalators don't break, they just temporarily become stairs. There should never be a sign that says 'Escalator Broken, Sorry for the Inconvenience'. It should say 'Escalator Temporarily Stairs, Sorry for the Convenience'"
 
-The data warehouse version of the sign is "Data warehouse temporarily has data from January 1, 2015 through two days ago, sorry for the convenience"
-
+The data warehouse version of the sign is "Data warehouse temporarily has data from the launch of our business through two days ago, sorry for the convenience"
 
 
 ### Monitoring process
-How do you implement & enforce your SLA?
+[WIP]
+
+
+### Issue communication
+You or one of your stakeholders noticed some of the data was off - how do you inform everyone who is impacted? The most efficient way to inform your stakeholders is through the platforms / methods they consume the data. If your stakeholders consume the data through an automated report emailed to the team every morning, reply all with an explanation of the issue and the estimated time to resolution. If your stakeholders consume the data through your business intelligence tool, display a warning at the top of BI tool (this is a very effective method of communication, but is not available on most systems).
+
+If the methods listed above are not an option and the data issue is severe (scope of impact or risk of making bad decisions) you have to over communicate. You have a duty to make sure your stakeholders are aware. Send a company-wide email or slack message and, because some people may not read your message, go around to individually speak with your stakeholders that are most likely to use the impacted data. 
 
-### Consequences
-Lose respect / partnership
+If it is a small data issue, you can probably get away with sending a message to your power users. If you use a "Data" Slack channel for your stakeholders to report data issues and ask data questions, you can use the same channel to communicate when the Data team identifies an issue. Otherwise, you can create a power user email distribution list for important announcements.
```

Comment:
I like this and agree. You  mentioned something similar to this, but do you also communicate what types of issues will be communicated to all in a formal way (e.g., email to org) vs. Slack @here?  This is sort of meta, but I find it helps with expectation setting.